src/openforms/emails/tests/test_tasks_integration.py

```diff
@@ -207,35 +208,41 @@ def test_email_sent_when_there_are_failures(self, mock_global_config, brk_config
             )
 
         with self.subTest("failed registration"):
-            admin_submissions_url = furl(
+            admin_relative_submissions_url = furl(
                 reverse("admin:submissions_submission_changelist")
             )
-            admin_submissions_url.args = {
+            admin_relative_submissions_url.args = {
                 "form__id__exact": form.id,
                 "needs_on_completion_retry__exact": 1,
                 "registration_time": "24hAgo",
             }
+            admin_submissions_url = build_absolute_uri(
+                admin_relative_submissions_url.url
+            )
```

Comment:
this just repeats the implementation details, can't you instead use furl to set up the expected state:

```py
admin_path = reverse("admin:submissions_submission_changelist")
admin_submissions_url = furl(f"http://testserver{admin_path}")
```

Reply:
Shouldn't I use `settings.BASE_URL` instead of the hardcoded url?This is what is taken into account in the method `admin_link`.

Reply:
well we control the value of the setting - you can make this more explicit with `@override_settings`, but in general indirection in tests is a bad practice.

Reply:
`testserver` is the default hostname for the django test client. It is not easily configurable, so it's fine to hardcode it in assertions.

Reply:
there is no test client, this runs in celery tasks without request context :)